vector/src/main/java/im/vector/listeners/ImageViewOnTouchListener.java

```diff
@@ -0,0 +1,123 @@
+package im.vector.listeners;
```

Comment:
Please add the copyright header to this new file. You can add your own name along with `New Vector` section